docs/treefmt-configuration.md

```diff
@@ -22,3 +22,12 @@ This section describes the integration between a single formatter and
 
 - `excludes`: A list of glob patterns to deny. If any of these patterns match,
   the file will be excluded.
+
+## Global config
+
+This section describes the config that applies to every formatters.
+In the top level of `treefmt.toml`:
```

Comment:
Do you mind moving these in a section? I noticed that top-level attributes are a bit awkward to use because the ordering matters. The section can be named `[global]` or something else if you have a preference.

Reply:
Good idea :+1: